tests/dsl/ptg/user-defined-functions/udf.jdf

```diff
@@ -65,7 +79,7 @@ UD_STARTUP1(m, n) [ startup_fn = ud_startup1 ]
 
 READ X <- A(m, n)
 
-BODY
+BODY [evaluate = %{ return PARSEC_HOOK_RETURN_DONE; %}]
```

Comment:
Did you intend to use `always_here` here? Or you want to use the inline (in which case we don't need to define `always_here` above)?

Reply:
I want to test all forms for defining this `evaluate` capability so I will keep the inline form here and use the `always_here` later for the UD_HASH_STRUCT CPU task.